definitions/ext-fastly_pop/fastly-pop_dashboard.json

```diff
@@ -293,26 +293,6 @@
             }
           }
         },
-        {
```

Comment:
Apart from removing some of them, it would be nice about reorganizing the rest of the visualizations being part of the layout.